src/testing/AddTestToIdentifierPage.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { FC } from 'react';
+import { Message, useTranslations } from 'retranslate';
+import { Box, Label, Heading, Container, Button, Alert, Input, Text } from 'theme-ui';
+import { useFormik } from 'formik';
+import * as yup from 'yup';
+
+import { CreateUserCommand, AuthenticationMethod } from '../api';
+import { useConfig } from '../common';
+import { useUserCreation } from './useUserCreation';
+
+const AnyBox = Box as any;
+
+// from https://ipsec.pl/data-protection/2012/european-personal-data-regexp-patterns.html
+const ESTONIAN_ID_CODE_REGEX = /^[1-6][0-9]{2}[1,2][0-9][0-9]{2}[0-9]{4}$/;
```

Comment:
I double-checked their regex and I think it's wrong — `[1,2]` is the first digit of `MM`, meaning it wouldn't allow `01`, for example 🤔

I'll replace with the `isikukood` package to have the same validation as the server.